src/YahtzeeGame.ts

```diff
@@ -1,12 +1,44 @@
+import {CategoryScorer, Chooser} from "./domain";
+import {Scorecard} from "./Scorecard";
+
 export class YahtzeeGame {
-  constructor() {
+
+  private runningScore: number;
+  private round: number;
+  private categoryScorer: CategoryScorer;
+  private chooser: Chooser;
+  private scorecard: Scorecard;
+
+  constructor(categoryScorer: CategoryScorer, chooser: Chooser, scorecard: Scorecard) {
+    this.round = 1;
+    this.chooser = chooser
+    this.categoryScorer = categoryScorer
+    this.scorecard = scorecard
+    this.runningScore = 0;
```

Comment:
I think this variable could be refactored out now that we introduced the Scorecard. The scorecard could have a `currentScore()` function or similar to sum up its internal state. No need to tack it separately in the game - in fact, doing so breaks domain boundaries.